UI/src/components/widgets/cloud/style.less

```diff
@@ -103,7 +103,7 @@ h3{
 }
 
 .widget-detail {
-  //width: 400px;
+  width: 400px;
   .widget-heading {
     font-family: 'Open Sans', sans-serif;
     font-size: 20px;
```

Comment:
Please dont override widget width here , it messes the widget width at other screens, as this is getting imported in index.html. if you want to do please add the style attribute to your html